frontend/src/app/shared/authentication.guard.spec.ts

```diff
@@ -21,13 +21,13 @@ describe('AuthenticationGuard', () => {
 
   it('should route if authenticated', async () => {
     authenticationService.isAuthenticated.and.returnValue(of(true));
-    const guardResult = await firstValueFrom(TestBed.runInInjectionContext(() => authenticationGuard(null, state)));
+    const guardResult = await lastValueFrom(TestBed.runInInjectionContext(() => authenticationGuard(null, state)));
     expect(guardResult).toBeTrue();
   });
 
   it('should login if not authenticated', async () => {
     authenticationService.isAuthenticated.and.returnValue(of(false));
-    const guardResult = await firstValueFrom(TestBed.runInInjectionContext(() => authenticationGuard(null, state)));
+    const guardResult = await lastValueFrom(TestBed.runInInjectionContext(() => authenticationGuard(null, state)));
```

Comment:
OOC why this change?

Reply:
I had a weird afterAll exception happening *sometimes*, and since that was the last thing I changed, I wondered if it came from the firstValueFrom. I don't thing that's it, but it doesn't change much.